Mathlib/MeasureTheory/Measure/FiniteMeasure.lean

```diff
@@ -240,9 +239,8 @@ theorem toMeasure_smul (c : R) (μ : FiniteMeasure Ω) : ↑(c • μ) = c • (
   rfl
 #align measure_theory.finite_measure.coe_smul MeasureTheory.FiniteMeasure.toMeasure_smul
 
-@[simp, norm_cast]
```

Comment:
Why can't this be `simp`?

Reply:
Because `FiniteMeasure` doesn't use `FunLike` and this coercion goes through `toMeasure`.